src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -67,8 +67,13 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
 
     @computed get geneSummary():JSX.Element {
         return (
-            <div>
+            <div style={{'paddingBottom':10}}>
```

Comment:
what's bottom padding for?

Reply:
@alisman  have some space before `Somatic Mutation Frequency`